modules/ROOT/nav.adoc

```diff
@@ -28,7 +28,9 @@ include::search:partial$nav.adoc[]
   ** xref:sdk:sdk-doctor.adoc[Troubleshooting Connections with SDK Doctor]
 
 include::n1ql:partial$nav.adoc[]
+
```

Comment:
Not sure about adding blank lines in the navigation list, they're fickle things, and the blank lines may have unwanted effects on the indenting of included partials. Have you done a preview build to check that it all works?

Reply:
I checked the navigation for Search and Query and everything works! I'll go ahead and merge this PR.

Preview here: https://preview.docs-test.couchbase.com/eventingnav/server/current/eventing/eventing-overview.html